lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -20,6 +20,8 @@ const pageFunctions = require('../../lib/page-functions.js');
  */
 /* istanbul ignore next */
 function runA11yChecks() {
+  /** Ignores the autofill information that is injected into the snippet via a chrome flag */
+  const snippetIgnoreAttrs = ['autofill-information', 'autofill-prediction', 'title'];
```

Comment:
Let's `.push` these values into `ignoreAttrs` inside `getOuterHTMLSnippet`, so no calling code has to handle this.

Reply:
so put the array directly in the getOuterHTMLSnippet function?

Reply:
https://github.com/gMakunde/lighthouse/pull/3